api/docs/v2/storefront/index.yaml

```diff
@@ -15,7 +15,7 @@ info:
       Please read our introduction to the API v2 [insert link here]
     </p>
     <p>
-      Click here to download Postman collection with example operations
+      <a href="https://raw.githubusercontent.com/spree/spree/master/api/docs/v2/storefront/index.yaml">Import this documentation to Postman</a>
```

Comment:
Maybe we could add `target="_blank" rel="noopener"` attributes? It'd make browsing docs more convenient.